data/DCL/projects.yaml

```diff
@@ -225,11 +219,9 @@ projects:
       - Info
     description: Scaling distributed Generative Adversarial Networks (GANs)
     layman_desc: >
-      The FeGAN system enables training GANs in the Federated Learning setup.
-      GANs are generative adversarial networks, a class of machine learning
-      where two neural networks contest with each other.
-      FeGAN is implemented on PyTorch and is using less bandwidth and is
-      faster than using stat-of-the-art GANs implementations.
+      The FeGAN system enables training GANs in the Federated Learning setup. GANs are generative adversarial networks, a class of machine
```

Comment:
Hmm - is the `layman_desc` needed? Because this is definitely not a layman description, but a `tech_desc`.

Also for the other `layman_desc` further down.

Reply:
As discussed, converted a lot of layman description (almost all) to tech desc when there is only one description